packages/sdk/src/across/clients/bobaBridge.ts

```diff
@@ -0,0 +1,101 @@
+import assert from "assert";
+import { ethers } from "ethers";
+import { Provider } from "@ethersproject/providers";
+import { Signer, BigNumber, ContractTransaction } from "ethers";
+import {
+  ERC20Ethers__factory,
+  OptimismL1StandardBridgeEthers__factory,
+  BobaAddressManagerEthers__factory,
+} from "@uma/contracts-node";
+import { Watcher } from "@eth-optimism/core-utils";
+
+export const l1Contracts: { ADDRESS_MANAGER_ADDRESS: { [chainId: number]: string } } = {
+  ADDRESS_MANAGER_ADDRESS: {
+    // mainnet
+    1: "0x8376ac6C3f73a25Dd994E0b0669ca7ee0C02F089",
+    // rinkeby
+    4: "0x93A96D6A5beb1F661cf052722A1424CDDA3e9418",
+  },
+};
+
+export class BobaBridgeClient {
```

Comment:
If this is very similar to Optimism, do you see an advantage of having them share code? Or do you think it's safer to keep them distinct in case they diverge in the future?